dashboard/app/dsl/contract_match_dsl.rb

```diff
@@ -8,7 +8,7 @@ def initialize
   def answer(text) @hash[:answers] << text end
 
   def i18n_strings
-    strings = super[@name]
+    strings = super [@name]
```

Comment:
I think this might actually be wrong?

`irb(main):001:0> class First
irb(main):002:1>   def foo
irb(main):003:2>     [1, 2, 3]
irb(main):004:2>   end
irb(main):005:1> end
=> :foo
irb(main):006:0> 
irb(main):007:0* class Nospace < First
irb(main):008:1>   def foo
irb(main):009:2>     super[1]
irb(main):010:2>   end
irb(main):011:1> end
=> :foo
irb(main):012:0> 
irb(main):013:0* class Space < First
irb(main):014:1>   def foo
irb(main):015:2>     super [1]
irb(main):016:2>   end
irb(main):017:1> end
=> :foo
irb(main):018:0> Nospace.new.foo
=> 2
irb(main):019:0> Space.new.foo
ArgumentError: wrong number of arguments (1 for 0)
	from (irb):2:in `foo'
	from (irb):15:in `foo'
	from (irb):19
	from /Users/laurel/.rbenv/versions/2.2.3/bin/irb:11:in `<main>'`